x-pack/plugins/security/public/views/management/edit_role/components/privileges/kibana/__snapshots__/impacted_spaces_flyout.test.tsx.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<ImpactedSpacesFlyout> renders without crashing 1`] = `
-<React.Fragment>
+<Fragment>
```

Comment:
question: Why do some of the snapshot differences change `<React.Fragment>` => `<Fragment>`, whereas others just remove the fragment altogether, [like this example](https://github.com/elastic/kibana/pull/24707/files#diff-cedb77dcc4a5fb2fae8b25b875e3adc0)?

Reply:
This is a good question. I looked into it a bit and couldn't find a solid answer. I looked at some of the source and nothing changed, it seemed like possibly when there is a single fragment inside of another single element, the fragment may disappear since it's "not needed" but I'm not totally sure if that's true.

Reply:
Enzyme has does this since roughly 3.3.5 or 3.3.6 - https://github.com/airbnb/enzyme/blame/master/packages/enzyme/src/RSTTraversal.js#L25 my question 

The fact that not all of the fragments go away makes me think there's multiple versions of react or enzyme or who knows what else in play.

Reply:
Dug in a bit more; in cases where the `Fragment` is the top-level/ROOT/HOST node (as it is in _ImpactedSpacesFlyout> renders without crashing 1_), then the Fragment is never seen as a child so it doesn't get expanded.